test/test_module_tracker.py

```diff
@@ -70,6 +75,7 @@ def forward(self, x):
             ],
         )
 
+    @skipIfTorchDynamo("unexplained 3.13+ recursion error")
```

Comment:
☠️.

Btw what's the preference of this v.s. `test/dynamo_expected_failures/...`?